packages/teams-js/test/promiseTester.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable-next-line recommend-const-enums/recommend-const-enums */
```

Comment:
Can you say more about why this one needs to be suppressed?

Reply:
See above